cmd/controller-manager/main.go

```diff
@@ -73,10 +73,10 @@ func parseFlags() {
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&certsDir, "certs", "/etc/webhook/certs",
 		"The directory for storing certs key file and cert file")
-	flag.StringVar(&configDir, "conf", "/etc/webhook/conf",
-		"The directory for storing webhook config files")
-	flag.StringVar(&watcherConfig.Namespace, "configmap-namespace", "",
+	flag.StringVar(&watcherConfig.Namespace, "template-namespace", "",
```

Comment:
This logic should be the same as the `configmap-namespace` we use like before? It will try to get the current namespace in its serviceaccount. Does this make sense to you?